portal-impl/test/unit/com/liferay/portal/util/PropsUtilTest.java

```diff
@@ -98,159 +88,10 @@ public void testGetLibDir() {
+			TestResult.class);
 
 		Assert.assertEquals(expectedPath, actualPath);
 	}
 
-	@Test
-	public void testGetPathURIFromURL() throws Exception {
-
-		// Tomcat
-
-		_testGetPathFromURL("jar:file:", "jar:file:/");
-		_testGetPathFromURL(
-			new URL(
-				"file:/opt/liferay/tomcat/classes/javax/servlet/Servlet.class"),
-			"/opt/liferay/tomcat/classes/javax/servlet/Servlet.class");
-		_testGetPathFromURL(
-			new URL(
-				"file:/C:/Liferay/tomcat/classes/javax/servlet/Servlet.class"),
-			"/C:/Liferay/tomcat/classes/javax/servlet/Servlet.class");
-
-		// Weblogic
-
-		_testGetPathFromURL("zip:", "zip:");
-
-		// Websphere
-
-		_testGetPathFromURL("wsjar:file:", "wsjar:file:/");
-		_testGetPathFromURL(
-			new URL(
-				"bundleresource://266.fwk-486185329/javax/servlet/Servlet." +
-					"class"),
-			"/javax/servlet/Servlet.class");
-
-		// Wildfly
-
-		_testGetPathFromURL("vfs:", "vfs:/");
-
-		// logging
-
-		try (LogCapture logCapture = LoggerTestUtil.configureJDKLogger(
-				PropsUtil.class.getName(), Level.FINE)) {
-
-			ReflectionTestUtil.invoke(
-				PropsUtil.class, "_getPathFromURL", new Class<?>[] {URL.class},
-				new URL(
-					"jar:file:/opt/liferay/tomcat/lib/servlet-api.jar" +
-						"!/javax/servlet/Servlet.class"));
-
-			List<LogEntry> logEntries = logCapture.getLogEntries();
-
-			Assert.assertEquals(logEntries.toString(), 1, logEntries.size());
-
-			LogEntry logEntry = logEntries.get(0);
-
-			Assert.assertEquals(
-				"URI file:/opt/liferay/tomcat/lib/servlet-api.jar!/javax" +
-					"/servlet/Servlet.class",
-				logEntry.getMessage());
-		}
-	}
-
-	@Test
-	public void testGetPathURIFromURLWithIllegalCharacter() {
-		try {
-			ReflectionTestUtil.invoke(
-				PropsUtil.class, "_getPathFromURL", new Class<?>[] {URL.class},
-				new URL(
-					"jar:file:/[opt/liferay/tomcat/lib/servlet-api.jar" +
-						"!/javax/servlet/Servlet.class"));
-
-			Assert.fail(
-				"SystemException caused by URISyntaxException should be " +
-					"thrown because of the illegal character '['");
-		}
-		catch (Exception exception) {
-			Assert.assertSame(SystemException.class, exception.getClass());
-
-			Throwable throwable = exception.getCause();
-
-			Assert.assertSame(URISyntaxException.class, throwable.getClass());
-		}
-	}
-
-	@Test
-	public void testGetPathURIFromURLWithUnknownProtocol() {
-		try {
-			ReflectionTestUtil.invoke(
-				PropsUtil.class, "_getPathFromURL", new Class<?>[] {URL.class},
-				new URL(
-					"jar", null, -1,
-					"unknown:/opt/liferay/tomcat/lib/servlet-api.jar!/javax" +
-						"/servlet/Servlet.class",
-					null));
-
-			Assert.fail(
-				"SystemException caused by MalformedURLException should be " +
-					"thrown because of the unknown protocol");
-		}
-		catch (Exception exception) {
-			Assert.assertSame(SystemException.class, exception.getClass());
-
-			Throwable throwable = exception.getCause();
-
-			Assert.assertSame(
-				MalformedURLException.class, throwable.getClass());
-		}
-	}
-
```

Comment:
Below is testing a private method that is removed for linux + windows protocol.
since I removed the method, theres no easy way to test these anymore. 
@vicnate5